.github/workflows/full-static.yml

```diff
@@ -45,8 +45,6 @@ jobs:
       with:
         go-version-file: go.mod
 
-    - name: build js
-      run: make build-web
```

Comment:
Is this safe for the time being? Or are the subsequent steps dependent on it?

Reply:
guessing yes, safe

the makefile will automatically run `build-web` as a prereq of `make full-static` below

the reason I split it was for clarity around timing

Reply:
Awesome, thanks for clarifying!